pkg/runner/local_docker.go

```diff
@@ -588,6 +588,11 @@ func ensureControlNetwork(ctx context.Context, cli *client.Client, log *zap.Suga
 		ctx,
 		log, cli,
 		"testground-control",
+		// making internal=false enables us to expose ports to the host (e.g.
+		// pprof and prometheus). by itself, it would allow the container to
+		// access the Internet, and therefore would break isolation, but since
+		// we have sidecar overriding the default Docker ip routes, and
+		// suppressing such traffic, we're safe.
```

Comment:
Added an explanation of why the flip from `internal=true` to `internal=false` is needed, and why it doesn't impact isolation. cc @coryschwartz